### ISSUE TITLE
SECURITY: Update weak SSH encryption

### ISSUE THREAD

**Issue by atxryan** [retaildevcrews/ngsa#489]
### Changes implemented

- waiting on scan to verify

- we need to update AKDC to address the below security issue and redeploy all VMs ASAP

- On the SSH server, disable RC4 and use secure ciphers. In the server's /etc/ssh/sshd_config, add the following: ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes192-ctr,aes128-ctr.

### This is already done on modern SSH clients
- On your local SSH client, enable the following ciphers: Host * ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes192-ctr,aes128-ctr.

**Comment by bartr** [comment]
PR merged to GitHub/Microsoft repo
